### PR TITLE
this, used as class member

### PR DESCRIPTION
**Pull request by deeper-x**

### CODE REVIEW

File: _examples/ThisClassMember.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: this class member
+---
+    
+**this** keyword, inside a class, refers to the instance. By default, used without identifiers, is referred to the current, innermost scope, the class where is declared.
```
Comment (hhariri) [inline _examples/ThisClassMember.md]: I think there are a few too many commas. 

By default when used without identifiers, **this** refers to the current innermost scope where the class is declared.

**Comment by deeper-x** [comment]
changed, according with your suggestions

**Comment by deeper-x** [comment]
Conflicts on _data/examples.yml solved.

**Comment by AlexanderPrendota** [comment]
Hi! Thank you for contributing. Please create the new PR and resolve all conflicts. Make sure you use new templates for examples.

Thanks! :)

**Pull request closed without merging**